scss/_utilities/_mediaqueries.scss

```diff
@@ -0,0 +1,43 @@
+/* 
+* Media Query Mixin  
+* Containes logic to easily remove media queries for
+* desktop only or mobile only projects
+* Examples:
+* 1. @include mq( 300px ) { } (min-width)
+* 2. @include mq( $max: 300px ) { } (max-width)
+* 3. @include mq( 400px, 800px ) { } (both min + max-width)
+*/
+$media-queries: true;
+$desktop-styles: true;
+
+@mixin mq($min: null, $max: null) {
+	@if $min == null and $max == null {
+		@error 'Please provide a value for $min and/or $max';
+	}
+
+	@if $media-queries {
+		@if $min != null and $max != null {
+			@media (min-width: $min) and (max-width: $max) {
+				@content;
+			}
+		}
+		@else if $min != null {
+			@media (min-width: $min) {
+				@content;
+			}
+		}
+		@else {
+			@media (max-width: $max) {
+				@content;
+			}
+		}
+	}
+	@else if $desktop-styles and $max == null {
+		/* Content printed with no media query for desktop project */
+		@content;
+	}
+	@else if $desktop-styles == false and $min == null {
```

Comment:
so say we have a max value passed and $desktop-styles is set to true? nothing happens ? So queries without a max value are considered as desktop styles  if I am not wrong? 
We could pass a third parameter to specify that explicitly maybe?